governance-and-automation/metadata-review/request-metadata-review.md

```diff
@@ -0,0 +1,43 @@
+---
+description: This article explains how to request a metadata review from collaborators. 
+---
+
+## Request Metadata Review
+
+Once you've [enabled the review option for your chosen metadata](manage-metadata-review.md), you can request a review by going to the **Sites Overview or Teams & Groups Overview**. 
```

Comment:
isto bi tu stavio više u smislu da ako imaš metadatu koju collaboratori mogu manage-irati, možeš zatražiti da to odrade kroz Point